src/ui/public/kuery/lib/filter_migration/lib/__tests__/exists.js

```diff
@@ -0,0 +1,35 @@
+import expect from 'expect.js';
+import { convertExistsFilter } from '../exists';
+
+describe('filter to kuery migration', function () {
+
+  describe('exists filter', function () {
+
+    it('should return a kuery node equivalent to the given filter', function () {
+      const filter = {
+        meta: {
+          type: 'exists',
+          key: 'foo',
+        }
+      };
+      const result = convertExistsFilter(filter);
+
+      expect(result).to.have.property('type', 'function');
+      expect(result).to.have.property('function', 'exists');
+      expect(result.arguments[0].value).to.be('foo');
+    });
+
+    it('should return undefined if the given filter is not of type "exists"', function () {
```

Comment:
But we don't have a type system :)   an exceptional scenario in one language is another language's bread and butter. Anyway, it doesn't matter much to me since this isn't a public API. I'll just switch them to exceptions since it'll be easier than adding a new function.